modules/govuk/manifests/apps/email_alert_service.pp

```diff
@@ -25,6 +25,9 @@
 # [*sentry_dsn*]
 #   The URL used by Sentry to report exceptions
 #
+# [*email_alert_api_bearer_token*]
+#   Bearer token for communication with the email-alert-api
+#
 class govuk::apps::email_alert_service(
```

Comment:
You'll need to add `email_alert_api_bearer_token` to the argument list. 😄

Reply:
Just testing to check if you're paying attention.